welcome/learn_more_about_openshift.adoc

```diff
@@ -30,44 +31,184 @@ Use the following sections to find content to help you learn about and use {prod
 |
+
+| xref:../networking/understanding-networking.adoc#understanding-networking[Manage networking]
+| xref:../networking/cluster-network-operator.adoc#cluster-network-operator[Cluster Network Operator], xref:../networking/multiple_networks/understanding-multiple-networks.adoc#understanding-multiple-networks[multiple network interfaces], and xref:../networking/network_security/network_policy/about-network-policy.adoc#about-network-policy[network policy]
+
+| xref:../operators/understanding/olm-understanding-operatorhub.adoc#olm-understanding-operatorhub[Manage Operators]
 |
 
+| xref:../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads_understanding-windows-container-workloads[Understanding Windows container workloads]
 |
-| xref:../backup_and_restore/index.adoc#backup-restore-overview[Backup and restore]
+
+|===
+
+[discrete]
+==== Changing cluster components
+
+[options="header",cols="2*"]
+|===
+|Learn more about changing cluster components |Additional resources
+
+|xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[Updating a cluster]
+| xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[Updating a cluster using the web console]; xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating using the CLI] and xref:../disconnected/updating/index.adoc#about-disconnected-updates[Using the OpenShift Update Service in a disconnected environment]
+
+| xref:../operators/understanding/crds/crd-extending-api-with-crds.adoc#crd-extending-api-with-crds[Use custom resource definitions (CRDs) to modify the cluster]
+| xref:../operators/understanding/crds/crd-extending-api-with-crds.adoc#crd-creating-custom-resources-definition_crd-extending-api-with-crds[create a CRD] and xref:../operators/understanding/crds/crd-managing-resources-from-crds.adoc#crd-managing-resources-from-crds[manage resources from CRDs]
+
+| xref:../applications/quotas/quotas-setting-per-project.adoc#quotas-setting-per-project[Set resource quotas]
+| xref:../applications/quotas/quotas-setting-per-project.adoc#quotas-setting-per-project[set quotas]
+
+| xref:../applications/pruning-objects.adoc#pruning-objects[Prune and reclaim resources]
+|
+
+| xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-infrastructure-practices.adoc#scaling-cluster-monitoring-operator[Scale] and xref:../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[tune] clusters
 |
+
+|===
+
+[id="observe-cluster"]
+== Observe a cluster
+
+[options="header",cols="2*"]
+|===
+|Learn about {product-title} |Additional resources
```

Comment:
Looks good. Olivia. Thanks! A lot of blank values exist in the Additional resources column so would it be best to put the Additional resources section under each table?

Reply:
Pointing to the feature docs is a better approach as against updating summary statements that might be subject to change.

Reply:
I am still adding in links, but even if some are blank I think we should still leave them. It gives us room to add more should we need to.

Reply:
OK. Maybe, "N/A" or "-" could be added to signify a blank entry. Sometimes bugs come in that a table entry is blank so the assumption is that something is missing.

Reply:
hmm yeah I can see what you are saying. @kalexand-rh could you weigh in on this since we talked about the empty table entries? I am not sure what the best option is. I don't think we should do `N/A` because there could be a good link, we just haven't identified it yet.